CHANGELOG.md

```diff
@@ -10,10 +10,12 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+- Fix IPv6 counting in stats ([#50])
```

Comment:
From my point of view everything (beside the recent egui addition) worked as intended ;)

```suggestion
- Count IPv4 and IPv6 statistics individually to avoid confusion ([#50])
```

Reply:
i guess it depends on who you ask :D

Reply:
Thanks!